ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -167,13 +174,15 @@ impl<'a> TraceBuilder<'a> {
                                     self.handle_type(self.aot_mod.type_(aot_field_ty))?;
                                 let load_ti_inst =
                                     jit_ir::LoadTraceInputInst::new(u32_off, input_tyidx).into();
-                                self.jit_mod.push(load_ti_inst)?;
-                                // If this take fails, we didn't see a corresponding store and the
-                                // IR is malformed.
-                                self.local_map.insert(
-                                    last_store_ptr.take().unwrap().to_inst_id(),
-                                    jit_ir::Operand::Local(self.jit_mod.last_inst_idx()),
-                                );
+                                if !is_sidetrace {
```

Comment:
Maybe a comment here explaining why we skip this step for side traces?

Reply:
[0e49706](https://github.com/ykjit/yk/pull/1287/commits/0e49706afb20e8ecef01903001133cd914b67e2e)